src/librustc_mir/dataflow/generic/engine.rs

```diff
@@ -167,124 +156,155 @@ where
         results
     }
 
-    /// Applies the cumulative effect of an entire block, excluding the call return effect if one
-    /// exists.
-    fn apply_whole_block_effect(
+    /// Helper function that propagates dataflow state into graph succesors until fixpoint is
+    /// reached.
+    fn _iterate_to_fixpoint(
```

Comment:
eek, I don't think you should name helpers with a *leading* underscore. Leading underscore is usually used for fields/methods that might be dead code.

I'd personally use a word to differentiate the helper routiine (e.g. `iterate_to_fixpoint_core`) but even just a *trailing* underscore would be better here than a leading underscore.

Reply:
This is my personal coding convention that probably originates from Python. I will switch this (as well as `_seek`) to a trailing underscore unless there is an established convention.